### ISSUE TITLE
Spigot Broke Things

### ISSUE THREAD

**Issue by mercurialmusic** [SydMontague/ImageMaps#9]
The plugin loads, but maps don't display unless they're replaced with the command. Chewing through map IDs like nobody's business. Blame random Spigot method changes? I'm also one Spigot version behind and it looks like they've done more on the Map API since.

Using ImageMaps 0.4.5 on CraftBukkit version git-Spigot-b0f4c22-3d19f01 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
```
[04:00:42] [Server thread/INFO]: [ImageMaps] Enabling ImageMaps v0.4.5
[04:00:42] [Server thread/ERROR]: Error occurred while enabling ImageMaps v0.4.5 (Is it up to date?)
java.lang.NoSuchMethodError: org.bukkit.Server.getMap(S)Lorg/bukkit/map/MapView;
	at de.craftlancer.imagemaps.ImageMaps.loadMaps(ImageMaps.java:226) ~[?:?]
	at de.craftlancer.imagemaps.ImageMaps.onEnable(ImageMaps.java:51) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:254) ~[spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:403) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:436) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:350) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:580) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:542) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:420) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:294) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:698) [spigot_latest.jar:git-Spigot-b0f4c22-3d19f01]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_162]
```

**Comment by SydMontague** [comment]
Thanks for the report, updating to the latest Spigot/CraftBukkit version should resolve the issue.

They indeed changed a method and had a bug in their compatibility layer that should prevent un-updated plugins from breaking, which has been fixed in git-Spigot-b0f4c22-c7ba979.

**Comment by mercurialmusic** [comment]
Yup! Sorry for the bother. Looks like the latest build did indeed fix it. I've been too used to them breaking things lately... :-/
